frontend/packages/client/src/pages/Proposal.js

```diff
@@ -379,7 +379,7 @@ export default function ProposalPage() {
       <section className="section">
         <div className="container">
           <WrapperResponsive extraClasses="mb-6" extraClassesMobile="mb-3">
-            <Link to={`/community/${proposal.communityId}`}>
+            <Link to={`/community/${proposal.communityId}?tab=proposals`}>
```

Comment:
this is for when coming back from the proposal page, to land again on the proposal tab instead of the about tab which is the first tab